src/async.tex

```diff
@@ -259,13 +259,13 @@
   &
 Shall not exit via an exception. \br\textit{Precondition:} A preceding call \tcode{x2.on_work_started()} where \tcode{x1 == x2}.  \\ \rowsep
 
-\tcode{x1.dispatch(std::move(f),a)}  &
+\tcode{x1.dispatch(\br\mbox{\ \ }std::move(f),~a)}  &
   &
 \effects Creates an object \tcode{f1} initialized with \tcode{\textit{DECAY_COPY}(forward<Func>(f))} (\CppXref{thread.decaycopy}) in the current thread of execution . Calls \tcode{f1()} at most once. The executor may block forward progress of the caller until \tcode{f1()} finishes execution. Executor implementations should use the supplied allocator to allocate any memory required to store the function object. Prior to invoking the function object, the executor shall deallocate any memory allocated. \enternote Executors defined in this Technical Specification always use the supplied allocator unless otherwise specified. \exitnote \br
 \sync The invocation of \tcode{dispatch} synchronizes with (\CppXref{intro.multithread}) the invocation of \tcode{f1}.  \\ \rowsep
 
-\tcode{x1.post(std::move(f),a)}\br
-\tcode{x1.defer(std::move(f),a)}  &
+\tcode{x1.post(std::move(f), a)}\br
+\tcode{x1.defer(std::move(f), a)}  &
```

Comment:
I guess I tried to keep the magic minimal, and this place didn't require non-breaking spaces. Note that semantically there's no reason not to break here; the only reason we do that in the place is to make *that* custom break work correctly. Coming to think of it, perhaps we don't even need the NBSP there, either.